### PR TITLE
Add locale support

### PR DESCRIPTION
**Pull request by TotalCaesar659**
Fixes https://github.com/ilyaigpetrov/copy-unicode-urls/issues/5.

### CODE REVIEW

File: _locales/ru/messages.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "extensionDescription": {
+    "message": "Копируйте ссылки Unicode в буфер обмена без ужасных кодировок с процентами или Punycode!",
+    "description": "Description of the extension."
+  },
+  "ifToDecode": {
+    "message": "Декодировать",
+    "description": ""
+  },
+  "ifToDecodeSentenceTerminators": {
```
Comment (ilyaigpetrov) [inline _locales/ru/messages.json:14]: It must be "ifToEncodeSentenceTerminators".

**Comment by ilyaigpetrov** [comment]
I couldn't find any obvious way of pushing my commits into this PR, so I pushed them into the `master`.
Your patch lacks translations of html pages, context menu items and the browser action title -- I added my translations.
Please, test your changes before making a PR next time (you may launch chrome in another language this way from a linux terminal: `LANGUAGE=ru google-chrome-stable`).
Version v0.0.11 with this changes is in the publishing review -- I will let you know in this thread when it's published.
Thank you for your contribution.
Donations are welcome too.

**Comment by ilyaigpetrov** [comment]
Changes are shipped with v0.0.11 in [the webstoe](https://chrome.google.com/webstore/detail/copy-unicode-urls/fnbbfiapefhkicjhecnoepbijhanpkjp).
Thank you.

**Comment by TotalCaesar659** [comment]
I tried to do my best but didn't go too far cause of my lack of skiils for it. Thank you very much for fixing my stuff and advices. I'll do more research if I want to add something in the future.

**Pull request closed without merging**